File: src/models/Interlocuteur.java
package models;


import java.io.Serializable;
import java.util.Date;
import javax.persistence.*;
@Entity
@Table(name="interlocuteur")
public class Interlocuteur implements Serializable {
	
    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    private Integer interid;

    @Column(name="fourid", columnDefinition="INTEGER", nullable=true)
    private Integer fourid ;
    
    @Column(name="utiid", columnDefinition="INTEGER", nullable=true)
    private Integer utiid ;
    
    @Column(name="cliid", columnDefinition="INTEGER", nullable=true)
    private Integer cliid ;
    
    @Column(name="internom", columnDefinition="VARCHAR(50)", nullable=true)
    private String internom ;
    
    @Column(name="interprenom", columnDefinition="VARCHAR(50)", nullable=true)
    private String interprenom ;
    
    @Column(name="intermail", columnDefinition="VARCHAR(50)", nullable=true)
    private String intermail ;
    
    @Column(name="intertel", columnDefinition="VARCHAR(20)", nullable=true)
    private String intertel ;
    
    @Column(name="interposte", columnDefinition="VRACHAR(100)", nullable=true)
    private String interposte ;
    
    @Column(name="interdteadd", columnDefinition="DATE", nullable=true)
    @Temporal(javax.persistence.TemporalType.DATE)
    private Date interdteadd ;
    
    @Column(name="intersuppr", columnDefinition="BOOLEAN", nullable=true)
    private boolean intersuppr ;
    
    @Column(name="interuniqid", columnDefinition="VARCHAR(500)", nullable=true)
    private String interuniqid ;

    public Integer getInterid() {
        return interid;
    }

    public void setInterid(Integer interid) {
        this.interid = interid;
    }

    public Integer getFourid() {
        return fourid;
    }

    public void setFourid(Integer fourid) {
        this.fourid = fourid;
    }

    public Integer getUtiid() {
        return utiid;
    }

    public void setUtiid(Integer utiid) {
        this.utiid = utiid;
    }

    public Integer getCliid() {
        return cliid;
    }

    public void setCliid(Integer cliid) {
        this.cliid = cliid;
    }

    public String getInternom() {
        return internom;
    }

    public void setInternom(String internom) {
        this.internom = internom;
    }

    public String getInterprenom() {
        return interprenom;
    }

    public void setInterprenom(String interprenom) {
        this.interprenom = interprenom;
    }

    public String getIntermail() {
        return intermail;
    }

    public void setIntermail(String intermail) {
        this.intermail = intermail;
    }

    public String getIntertel() {
        return intertel;
    }

    public void setIntertel(String intertel) {
        this.intertel = intertel;
    }

    public String getInterposte() {
        return interposte;
    }

    public void setInterposte(String interposte) {
        this.interposte = interposte;
    }

    public Date getInterdteadd() {
        return interdteadd;
    }

    public void setInterdteadd(Date interdteadd) {
        this.interdteadd = interdteadd;
    }

    public boolean isIntersuppr() {
        return intersuppr;
    }

    public void setIntersuppr(boolean intersuppr) {
        this.intersuppr = intersuppr;
    }

    public String getInteruniqid() {
        return interuniqid;
    }

    public void setInteruniqid(String interuniqid) {
        this.interuniqid = interuniqid;
    }
    
}

File: application/models/Interlocuteur.php
<?php

/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Interlocuteur
 *
 * @author overadmin
 */
class Application_Model_Interlocuteur  extends Application_Model_Element_Abstract{    
    
    protected $idField = 'interid';
    
    protected $fourid;
    
    protected $cliid;
    
    protected $internom;
    
    protected $intermail;
    
    protected $intertel;
    
    protected $intersuppr;
    
    protected $interprenom;
    
    protected $_utiid;
    
    protected $_interposte;
    
    protected $_interdteadd;
    
    public function getInterdteadd() {
        return $this->_interdteadd;
    }

    public function setInterdteadd($interdteadd) {
        $this->_interdteadd = $interdteadd;
    }

        
    public function getInterposte() {
        return $this->_interposte;
    }

    public function setInterposte($interposte) {
        $this->_interposte = $interposte;
    }

    
    public function getUtiid() {
        return $this->_utiid;
    }

    public function setUtiid($utiid) {
        $this->_utiid = $utiid;
    }

        public function getIdField() {
	return $this->idField;
    }

    public function setIdField($idField) {
	$this->idField = $idField;
    }

    public function getFourid() {
	return $this->fourid;
    }

    public function setFourid($fourid) {
	$this->fourid = $fourid;
    }

    public function getCliid() {
	return $this->cliid;
    }

    public function setCliid($cliid) {
	$this->cliid = $cliid;
    }

    public function getInternom() {
	return $this->internom;
    }

    public function setInternom($internom) {
	$this->internom = $internom;
    }

    public function getIntermail() {
	return $this->intermail;
    }

    public function setIntermail($intermail) {
	$this->intermail = $intermail;
    }

    public function getIntertel() {
	return $this->intertel;
    }

    public function setIntertel($intertel) {
	$this->intertel = $intertel;
    }

    public function getIntersuppr() {
	return $this->intersuppr;
    }

    public function setIntersuppr($intersuppr) {
	$this->intersuppr = $intersuppr;
    }   
    
    public function getInterprenom() {
	return $this->interprenom;
    }

    public function setInterprenom($interprenom) {
	$this->interprenom = $interprenom;
    }
}

?>
